src/charts/c3/c3.mixin.js

```diff
@@ -0,0 +1,57 @@
+angular.module('patternfly.charts').factory('ChartsMixin', function() {
+    'use strict';
+
+    // Donut Chart Defaults
+    var getDefaultDonut = function(title) {
+        return {
+            title: title,
+            label: {
+                show: false
+              },
+              width: 10
+            };
+      };
+
+    var getDefaultDonutSize = function() {
```

Comment:
Sorry, what I mean is this:

```javascript
angular.module('patternfly.charts').factory('ChartsMixin', function() {
  var getDefaultDonut, getDefaultDonutSize;

  getDefaultDonut = function () {...};
  getDefaultDonutSize = function () {...};

};
```

This is actually equivalent to what you have currently once it's parsed.  The trouble is that by not declaring your `var`s up top you can run into oddities like this

```javascript

var blah = 1;
function someFunction () {
	if (!blah) {
		var blah = 10;
	}
	console.log(blah);
}
someFunction();
```

This results in 10.  Now this is obviously a contrived example but it's a best practice to declare `var`s up top, outright to avoid any funny business (javascript is already funny enough).